### ISSUE TITLE
Nuxt 3 

### ISSUE THREAD

**Issue by productdevbook** [nuxt-community/apollo-module#413]
Will there be an update when nuxt 3 comes out?

**Comment by chaitb** [comment]
Any plans to support Nuxt3, as it has been released in Public Beta now?

**Comment by ruslansteiger** [comment]
You can check the status here: https://github.com/nuxt/framework/discussions/751

**Comment by mfreeman451** [comment]
Any comments about this, I know it is being tracked but that doesn't tell us anything..

**Comment by productdevbook** [comment]
example apollo vue used. https://github.com/productdevbook/productdevbook.com

I'm still in the development phase.

**Comment by HelloAlexPan** [comment]
Has there been any updaters here? Our company is starting a fresh project on Nuxt 3 and have the capacity to help work on a Nuxt 3 build if there are any tasks that need to be completed

**Comment by PTiCA1** [comment]
Any update? Or info to update to apollo v4?

**Comment by yacosta738** [comment]
Any update or info to support Nuxt 3

![bitmoji](https://sdk.bitmoji.com/render/panel/372c1889-7dd9-4082-addd-1c963a8ec557-db1ffc10-9e7f-4bea-a043-e744f59d5043-v1.png?transparent=1&palette=1&width=246)

**Comment by bp-janelle** [comment]
Any movement on this?

**Comment by dargmuesli** [comment]
Guys and girls, I'll personally make sure to have you pinged once there is progress. You can refrain from asking once more with all available peace of mind, avoiding pings for anyone interested.

**Comment by dargmuesli** [comment]
Here it is, the promised ping:
I migrated the module to Apollo client v3 and updated all dependencies to their most recent versions. That migration was successful per se, but SSR failed. The server sent the correct result, then the client detected a diff to its state, forced a refetch and got stuck in the loading state. Fun, and frustrating. After lots of research, I found a couple of likely matching issues on GitHub [1] [2] that are open for 2 or 3 years now. It sounds like it could work for React, but we're speaking about Vue here.

It seems like @Diizzayy found another way, with a PR coming though! We'll keep you posted.
Whoever's up for testing that PR, feel free to give a ping back! :)

[1] https://github.com/vuejs/apollo/issues/585
[2] https://github.com/apollographql/apollo-client/issues/6631
